From ad558ca953b635f4491973cfe63b6d78cb696f1c Mon Sep 17 00:00:00 2001
From: dghart <dghart@users.sourceforge.net>
Date: Sun, 13 Feb 2022 14:12:51 +0000
Subject: [PATCH] Fix a wxAssert when showing the Command-line terminal

"Must have wxTE_PROCESS_ENTER for wxEVT_TEXT_ENTER to work", though it seemed fine without it :/
---
 Tools.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Tools.h b/Tools.h
index e01a97f..e1beb36 100644
--- a/Tools.h
+++ b/Tools.h
@@ -378,7 +378,7 @@ class TerminalEm : public TextCtrlBase    // Derive from this so that (in gtk2)
 public:
 TerminalEm(){};
 TerminalEm(wxWindow* parent, wxWindowID id, const wxString& value = wxT(""), bool multline = true,  const wxPoint& pos = wxDefaultPosition,
-           const wxSize& size = wxDefaultSize, long style = 0, const wxValidator& validator = wxDefaultValidator,  const wxString& name = wxT("TerminalEm"))
+           const wxSize& size = wxDefaultSize, long style = wxTE_PROCESS_ENTER, const wxValidator& validator = wxDefaultValidator,  const wxString& name = wxT("TerminalEm"))
             :     TextCtrlBase(parent, id, value, multline, pos, size, style, validator, name), multiline(multline) { Init(); }
 ~TerminalEm();
 void Init();                                               // Do the ctor work here, as otherwise wouldn't be done under xrc
-- 
2.37.2

